src/legacy/server/logging/log_format_string.js

```diff
@@ -70,12 +70,23 @@ export default class KbnLoggerStringFormat extends LogFormat {
     const time = color('time')(this.extractAndFormatTimestamp(data, 'HH:mm:ss.SSS'));
     const msg = data.error ? color('error')(data.error.stack) : color('message')(data.message);
 
-    const tags = _(data.tags)
-      .sortBy(function (tag) {
-        if (color(tag) === _.identity) return `2${tag}`;
-        if (_.includes(statuses, tag)) return `0${tag}`;
-        return `1${tag}`;
-      })
```

Comment:
So basically, every meta informations where put in the same `tags` list, and that's the way we were trying to reorder them the best we can ? That seems like a poor implementation.

However just to understand, the mechanism did not change between legacy and NP, so I'm not sure why we only encounter the issue in NP ? Is that because in legacy we basically didn't have multiple non-status tags, so that no more than one tag was sorted by `return `1${tag}`;` ?